docker/main/ngen/Dockerfile

```diff
@@ -652,6 +654,8 @@ ARG BUILD_TOPMODEL
 ARG BUILD_PET
 ARG BUILD_SLOTH
 
+ARG NUMPY_CONSTRAINT
```

Comment:
This is an abuse of the args parameter. Do not use this to inject python versions. Stick to setup.py and requirements.txt.